frontend/src/assets/css/variables.scss

```diff
@@ -0,0 +1,5 @@
+:root {
```

Comment:
import this file in `frontend/src/assets/css/styles.scss` and it will be accessed globally